src/png.imageio/png_pvt.h

```diff
@@ -558,8 +558,9 @@ put_parameter(png_structp& sp, png_infop& ip, const std::string& _name,
 
 
 /// Writes PNG header according to the ImageSpec.
+/// \return empty string on success, error message on failure.
 ///
-inline void
+inline const std::string
```

Comment:
Mostly for my own curiosity: what purpose does the `const` serve here?

Reply:
This `const` is in every other functions returning an error `std::string`, so I kept it to keep a consistent code-style.